testplan/testing/multitest/entries/assertions.py

```diff
@@ -122,6 +123,25 @@ def evaluate(self):
 
 
 class Fail(Assertion):
+    def __init__(
+        self, description=None, category=None, flag=None, message=None
```

Comment:
To achieve the same behavior with result.log, shall we have 'message' as the first and required parameter?